views/home.hbs

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Firefox Breach Alerts</title>
+        <link rel="stylesheet" href="/css/foundation.min.css">
+        <script type="text/javascript" src="/js/app.js" async></script>
+        <script type="text/javascript" src="/js/vendor/jquery.js" defer></script>
+        <script type="text/javascript" src="/js/vendor/foundation.min.js" defer></script>
+    </head>
+
+    <body>
+        <div class="top-bar">
+          <div class="top-bar-left">
+            <h1><a href="/">Firefox Breach Alert</a></h1>
+          </div>
+          <div class="top-bar-right">
+            <ul class="menu">
+              <li><a href="/">About</a></li>
+              <li><a href="#tips">Tips</a></li>
```

Comment:
nit: there is no DOM element with an `id` of "tips" (or anchor name). But again, I'm happy to file as a separate bug after this lands (since it's something I'll probably end up forgetting to check in the future unless I remind myself).